Scripts/SL_Init.lua

```diff
@@ -378,6 +378,21 @@ SL = {
 			InitialValue=0.5,
 		},
 	},
+	ExWeights = {
+		-- W0 is not necessarily a "real" window.
+		-- In ITG mode it is emulated based off the value of TimingWindowW1 defined
+		-- for FA+ mode.
+		W0=3.5,
+		W1=3,
+		W2=2,
+		W3=1,
+		W4=0,
+		W5=0,
+		Miss=0,
+		LetGo=0,
+		Held=3.5,  -- Should be same as W0 as that is how SM handles it internally.
+		HitMine=-1
```

Comment:
Not exactly sure how they work, but does this also take rolls into account?

Reply:
Holds and Rolls implicitly have the same weight (this is consistent with the SM engine).
```ini
PercentScoreWeightW1=5,
PercentScoreWeightW2=4,
PercentScoreWeightW3=2,
PercentScoreWeightW4=0,
PercentScoreWeightW5=-6,
PercentScoreWeightMiss=-12,
PercentScoreWeightLetGo=0,
PercentScoreWeightHeld=IsGame("pump") and 0 or 5,
PercentScoreWeightHitMine=-6,
PercentScoreWeightCheckpointHit=0, <-- technically not doing anything with this but /shrug
```